### PR TITLE
Merge `master` --> `netcore`

### PR DESCRIPTION
**Pull request by adamchester**
* Mono.Cecil 0.10-beta5
 * Paket 4.1.7

### CODE REVIEW

**Comment by adamchester** [comment]
Hey @haf, what [this](https://ci.appveyor.com/project/haf/expecto/build/0.0.1.121#L20) mean?

```
Build started
git clone -q https://github.com/haf/expecto.git C:\projects\expecto
git fetch -q origin +refs/pull/137/merge:
git checkout -qf FETCH_HEAD
Running Install scripts
bundle install
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/.
Resolving dependencies...
Installing rake 10.5.0
Installing map 6.6.0
Installing mini_portile2 2.1.0
Installing semver2 3.4.2
Using bundler 1.14.5
Installing nokogiri 1.7.1 (x86-mingw32)
Gem::InstallError: nokogiri requires Ruby version < 2.4, >= 2.1.0.
An error occurred while installing nokogiri (1.7.1), and Bundler cannot
continue.
Make sure that `gem install nokogiri -v '1.7.1'` succeeds before bundling.
Command exited with code 5
```

**Comment by haf** [comment]
You need this commit from master https://github.com/haf/expecto/commit/12c7af06dd6760f9b6e14f910cd5020fc51bf12e

**Comment by adamchester** [comment]
Didn't I get that [her](https://github.com/haf/expecto/pull/137/files#diff-180360612c6b8c4ed830919bbb4dd459R9)?

File: appveyor.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ environment:
 
 install:
   - bundle install
+  - set PATH=C:\Ruby23-x64\bin;%PATH%
```
Comment (haf) [inline appveyor.yml]: No, you put it below its usage (above)

Comment (adamchester) [inline appveyor.yml]: oops... fixed thanks 👍

**Comment by haf** [comment]
I can build and test on OS X well enough; only merging of nuget packages left!

**Comment by haf** [comment]
I'm not sure how to do assembly redirects with dotnet – paket is normally what I use?

**Comment by haf** [comment]
I tried bumping nugets https://github.com/haf/expecto/tree/adamchester-netcore and got

screen shot 2017-04-14 at 17 41 28

**Comment by adamchester** [comment]
Hey @haf.

I grabbed [adamchester-netcore](https://github.com/haf/expecto/tree/adamchester-netcore) and made these changes:

```diff
diff --git a/Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj b/Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj
index 01511ec..cdd664e 100755
--- a/Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj
+++ b/Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj
@@ -31,5 +31,5 @@
     <ProjectReference Include="../Expecto.netcore/Expecto.netcore.fsproj" />
     <PackageReference Include="FsCheck" Version="3.0.0-alpha-b752" />
   </ItemGroup>
-  <Import Project="..\.paket\Paket.Restore.targets" />
+  <!--<Import Project="..\.paket\Paket.Restore.targets" />-->
 </Project>
\ No newline at end of file
diff --git a/Expecto.netcore/Expecto.netcore.fsproj b/Expecto.netcore/Expecto.netcore.fsproj
index 87d739b..2d8de72 100644
--- a/Expecto.netcore/Expecto.netcore.fsproj
+++ b/Expecto.netcore/Expecto.netcore.fsproj
@@ -18,5 +18,5 @@
     <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
     <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.3.0" />
   </ItemGroup>
-  <Import Project="..\.paket\Paket.Restore.targets" />
+  <!--<Import Project="..\.paket\Paket.Restore.targets" />-->
 </Project>
\ No newline at end of file
```

That allowed it to build. Do you get the same result?

**Comment by adamchester** [comment]
see #140

**Pull request closed without merging**